### PR TITLE
wallet dashboard

### PR DESCRIPTION
**Pull request by M-Rb3**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration(if any)**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: components/dashboard/wallet/Balance.js [hunk 1]
```diff
@@ -25,26 +25,29 @@ const CurrencyBalances = (props) => {
...
       {/* title row */}
-      <div
+      {/* <div
         className="d-flex flex-row justify-content-between align-items-start"
         style={{ height: '15%', width: '100%', marginBottom: '15px' }}
       >
         <p style={{ fontSize: '20px', fontWeight: '700', color: 'white' }}>
           Currency Balances
         </p>
         <button type="button">Add Token</button>
```
Comment (Ebube111) [inline components/dashboard/wallet/Balance.js:35]: If these code are no longer needed in the future then i suggest cleanup is needed or WDYT

File: components/dashboard/wallet/coinsList.js [hunk 2]
```diff
@@ -36,59 +39,76 @@ export default [
...
+  //   percent_change_30d: 4.73,
+  //   color: 'orange',
+  //   volume_24h: 26140524526.52,
+  // },
+  // {
+  //   id: 5,
+  //   name: 'Bitcoin Cash',
+  //   symbol: 'BCH',
+  //   price: 9798.977911,
+  //   price_btc: 1,
```
Comment (Ebube111) [inline components/dashboard/wallet/coinsList.js:80]: same as this page, if the code is not needed for later then i suggest cleanup

**Review by Ebube111 — COMMENTED** [review]
Generally the overall code looks good to me. Nice Job

**Comment by ltran0817** [comment]
Since this dashboard was made for the light theme, the loading skeleton color I used was gray. For dark theme I suggest to use some light purple color to match it, or can use black since it goes well with everything. Other than that everything looks amazing to me!

**Comment by M-Rb3** [comment]
image
route '/wallet'

**Pull request merged**